types/event_record.go

```diff
@@ -81,7 +81,71 @@ type EventRecords struct {
+	Recovery_RecoveryRemoved           []EventRecoveryRemoved                   //nolint:stylecheck,golint
+	Sudo_Sudid                         []EventSudoSudid                         //nolint:stylecheck,golint
+	Sudo_KeyChanged                    []EventSudoKeyChanged                    //nolint:stylecheck,golint
+	Sudo_SudoAsDone                    []EventSudoAsDone                        //nolint:stylecheck,golint
+	Council_Proposed                   []EventCollectiveProposed                //nolint:stylecheck,golint
+	Council_Voted                      []EventCollectiveProposed                //nolint:stylecheck,golint
+	Council_Approved                   []EventCollectiveApproved                //nolint:stylecheck,golint
+	Council_Disapproved                []EventCollectiveDisapproved             //nolint:stylecheck,golint
+	Council_Executed                   []EventCollectiveExecuted                //nolint:stylecheck,golint
+	Council_MemberExecuted             []EventCollectiveMemberExecuted          //nolint:stylecheck,golint
+	Council_Closed                     []EventCollectiveClosed                  //nolint:stylecheck,golint
+	Treasury_Proposed                  []EventTreasuryProposed                  //nolint:stylecheck,golint
+	Treasury_Spending                  []EventTreasurySpending                  //nolint:stylecheck,golint
+	Treasury_Awarded                   []EventTreasuryAwarded                   //nolint:stylecheck,golint
+	Treasury_Rejected                  []EventTreasuryRejected                  //nolint:stylecheck,golint
+	Treasury_Burnt                     []EventTreasuryBurnt                     //nolint:stylecheck,golint
+	Treasury_Rollover                  []EventTreasuryRollover                  //nolint:stylecheck,golint
 	Treasury_Deposit                   []EventTreasuryDeposit                   //nolint:stylecheck,golint
+	Treasury_NewTip                    []EventTreasuryNewTip                    //nolint:stylecheck,golint
+	Treasury_TipClosing                []EventTreasuryTipClosing                //nolint:stylecheck,golint
+	Treasury_TipClosed                 []EventTreasuryTipClosed                 //nolint:stylecheck,golint
+	Treasury_TipRetracted              []EventTreasuryTipRetracted              //nolint:stylecheck,golint
+	Utility_BatchInterrupted           []EventUtilityBatchInterrupted           //nolint:stylecheck,golint
+	Utility_BatchCompleted             []EventUtilityBatchCompleted             //nolint:stylecheck,golint
+	Utility_NewMultisig                []EventUtilityNewMultisig                //nolint:stylecheck,golint
+	Utility_MultisigApproval           []EventUtilityMultisigApproval           //nolint:stylecheck,golint
+	Utility_MultisigExecuted           []EventUtilityMultisigExecuted           //nolint:stylecheck,golint
+	Utility_MultisigCancelled          []EventUtilityMultisigCancelled          //nolint:stylecheck,golint
```

Comment:
Would you also add the events for our multi account pallet?

It technically does not belong into GSRPC, but since we are maintaining both projects ;)

Reply:
I disagree, it has to come from the client.

Reply:
Fine for me :)